src/sentry/replays/endpoints/project_replay_clicks_index.py

```diff
@@ -41,14 +47,43 @@
 REFERRER = "replays.query.query_replay_clicks_dataset"
 
 
+class ReplayClickResponseData(TypedDict):
+    node_id: int
+    timestamp: str
+
+
+class ReplayClickResponse(TypedDict):
+    data: list[ReplayClickResponseData]
+
+
 @region_silo_endpoint
+@extend_schema(tags=["Replays"])
 class ProjectReplayClicksIndexEndpoint(ProjectEndpoint):
     owner = ApiOwner.REPLAY
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
     }
 
+    @extend_schema(
+        operation_id="Retrieve a collection of RRWeb DOM node-ids and the timestamp they were clicked.",
```

Comment:
let's shorten (and capitalize) this title -- it'll show up in the sidebar like this:
<img width="221" alt="SCR-20240208-hyox" src="https://github.com/getsentry/sentry/assets/56095982/e4992edd-cb35-48a2-9bd9-d1961a03f10f">